src/ts/component/menu/dataview/calendar.tsx

```diff
@@ -94,11 +94,27 @@ const MenuCalendar = observer(class MenuCalendar extends React.Component<I.Menu>
 						return (
 							<div 
 								key={i} 
+								id={`day-${item.y}-${item.m}-${item.d}`}
```

Comment:
`{`day-${item.y}-${item.m}-${item.d}`}` maybe change to
`[ 'day', item.y, item.m, item.d ].join('-')`